packages/validator/src/lib/PwaValidator.ts

```diff
@@ -95,22 +94,20 @@ export class PwaValidator {
     const accessibilityScore = psiResult.lighthouseResult.categories.accessibility.score;
 
     // Web Vitals Scores
-    const fcpScore =
-      psiResult.lighthouseResult.audits.metrics.details.items[0]['firstContentfulPaint'];
+    const lighthouseAuditMetrics = psiResult.lighthouseResult.audits.metrics.details.items[0];
+    const fcpScore = lighthouseAuditMetrics.firstContentfulPaint;
+    const roundedFcpScore = PwaValidator.roundToNearestMultiple(fcpScore, 100);
```

Comment:
Is "`PwaValidator.`" needed here?

Reply:
It is, as it's a static method in PwaValidator.